appveyor.yml

```diff
@@ -40,35 +40,36 @@ build_script:
   - SET SWIG=0 # disables swig for mkn
   - call "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Auxiliary\Build\vcvars64.bat"
 
+  # doesnt exist for somereason?
+  - mkdir -p "C:\Program Files (x86)\Windows Kits\10\include\10.0.16299.0\cppwinrt"
+
   # Python 3.6
   - cat appveyor/pip/numpy/numpy-1.14.0+mkl-cp36* > appveyor/pip/numpy-1.14.0+mkl-cp36-cp36m-win_amd64.whl
   - C:\Python36-x64\Scripts\pip install appveyor\pip\numpy-1.14.0+mkl-cp36-cp36m-win_amd64.whl
   - C:\Python36-x64\Scripts\pip install appveyor\pip\numpydoc-0.7.0-py2.py3-none-any.whl
   - C:\Python36-x64\Scripts\pip install appveyor\pip\scipy-1.0.0-cp36-cp36m-win_amd64.whl
   - C:\Python36-x64\Scripts\pip install -r requirements.txt
   - C:\Python36-x64\Scripts\pip install tensorflow
-  - C:\Python36-x64\python setup.py build_ext --inplace 
+  - C:\Python36-x64\python setup.py build_ext --inplace
   - C:\Python36-x64\python -m unittest discover -v . "*_test.py"
-  - C:\Python36-x64\python setup.py bdist_wheel
   - bash -c "PATH=\"$PWD:/c/Python36-x64/:/c/Program Files/Git/usr/bin:$PATH\"; sh/mkn.sh"
-  - bash -c "PATH=\"$PWD:/c/Python36-x64/:/c/Program Files/Git/usr/bin:$PATH\"; sh/gtest.sh" 
+  - bash -c "PATH=\"$PWD:/c/Python36-x64/:/c/Program Files/Git/usr/bin:$PATH\"; sh/gtest.sh"
+  - C:\Python36-x64\python setup.py bdist_wheel
   - rm -rf build
   - rm -rf lib\bin
 
-  # Python 3.5
-  - cat appveyor/pip/numpy/numpy-1.14.0+mkl-cp35* > appveyor/pip/numpy-1.14.0+mkl-cp35-cp35m-win_amd64.whl
-  - C:\Python35-x64\Scripts\pip install appveyor\pip\numpy-1.14.0+mkl-cp35-cp35m-win_amd64.whl
-  - C:\Python35-x64\Scripts\pip install appveyor\pip\numpydoc-0.7.0-py2.py3-none-any.whl
-  - C:\Python35-x64\Scripts\pip install appveyor\pip\scipy-1.0.0-cp35-cp35m-win_amd64.whl
-  - C:\Python35-x64\Scripts\pip install -r requirements.txt
-  - C:\Python35-x64\Scripts\pip install tensorflow
-  - C:\Python35-x64\python setup.py build_ext --inplace 
-  - C:\Python35-x64\python -m unittest discover -v . "*_test.py"
-  - C:\Python35-x64\python setup.py bdist_wheel
-  - bash -c "PATH=\"$PWD:/c/Python35-x64/:/c/Program Files/Git/usr/bin:$PATH\"; sh/mkn.sh"
-  - bash -c "PATH=\"$PWD:/c/Python35-x64/:/c/Program Files/Git/usr/bin:$PATH\"; sh/gtest.sh" 
-  - rm -rf build
-  - rm -rf lib\bin
+  # # Python 3.5 # with addition of gtests appveyor is taking more than an hour and timing out
```

Comment:
Is this still true now that we don't launch all possible combinations?

Reply:
it's uncertain at the moment

Reply:
I will return to this in future and assess - we might want to remove the C++ serialization gtests entirely as it's pretty much done in python